src/core/store/modules/lines/lines.reducer.ts

```diff
@@ -240,6 +250,20 @@ const linesReducer = createReducer(linesInitialState, (builder) => {
       state.statusMap.user.getBorrowerPositions = { error: error.message };
     })
 
+    /* ------------------------- getUserPortfolio ------------------------- */
+    .addCase(getUserPortfolio.pending, (state, { meta }) => {
+      state.statusMap.user.getUserPortfolio = { loading: true };
+    })
+    .addCase(getUserPortfolio.fulfilled, (state, { meta, payload: { userPortfolio } }) => {
+      console.log('User Portfolio: positions in reducer', userPortfolio);
```

Comment:
@Seroxdesign I left this console.log in so you could see the data that flows through to the Portfolio component.

Reply:
![image](https://user-images.githubusercontent.com/69639595/205651014-71937ab1-0f9b-40ea-988d-6c5f762c39a1.png)

Seems like it's returning an error